diff --git a/highs/HConfig.h.bazel.in b/highs/HConfig.h.bazel.in
index 456229e7e..e10b7f2a2 100644
--- a/highs/HConfig.h.bazel.in
+++ b/highs/HConfig.h.bazel.in
@@ -17,9 +17,9 @@
 #define HIGHS_HAVE_BITSCAN_REVERSE
 #endif
 
-#define HIGHS_GITHASH "bdf95f2e0"
+#define HIGHS_GITHASH "755a8e027"
 #define HIGHS_VERSION_MAJOR 1
-#define HIGHS_VERSION_MINOR 9
+#define HIGHS_VERSION_MINOR 12
 #define HIGHS_VERSION_PATCH 0
 /* #undef HIGHS_DIR */
 
